deploy/v2/python/downloader/SAP_DLM.py

```diff
@@ -152,14 +152,14 @@ def download(self):
 
         # Second request to download the file from new or resume
         while True:
-            resume_header = ({'Range': 'bytes={self.last_pos}-'})
+            resume_header = ({'Range': 'bytes=%d-' % self.last_pos})
             resp  = DLM.sess.get(DLM.url_token, params=payload, timeout=resp_timeout_sec, stream=True, headers=resume_header)
             if self.last_pos > 0:
                 print("Resume at last_pos %s" % self.last_pos)
-            if resp.status_code == 200:
+            if resp.status_code in [200, 206]:
```

Comment:
```suggestion
            if resp.status_code >= 200 and resp.status_code < 300:
```
(to allow other successful HTTP responses)